backend-log-o-meme/db/Seeder.rb

```diff
@@ -1,33 +1,65 @@
 require 'sqlite3'
-require 'faker'
+# require 'faker'
 require 'bcrypt'
+require 'pp'
 class Seeder
 
   def self.seed! 
-    db = SQLite3::Database.new 'db/todos.db'
-    db.execute 'DROP TABLE IF EXISTS todos'
-    db.execute 'CREATE TABLE todos (
-                  "id"	INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
-                  "title" VARCHAR(255) NOT NULL,
-                  "description" TEXT,
-                  "isCompleted" BOOLEAN DEFAULT 0,
-                  "user_id" INTEGER NOT NULL)'
-    
+    db = SQLite3::Database.new 'db/database.db'
+
     db.execute 'DROP TABLE IF EXISTS users'
+    db.execute 'DROP TABLE IF EXISTS comments'
+    db.execute 'DROP TABLE IF EXISTS logs'
+    
+
     db.execute 'CREATE TABLE users (
+                  "id"	INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+                  "email" VARCHAR(255) UNIQUE NOT NULL,
+                  "name" TEXT NOT NULL,
+                  "password_hash" TEXT NOT NULL,
+                  "permisions" INTEGER NOT NULL,
+                  "mentor_id" INTEGER)'
+
+    db.execute 'CREATE TABLE comments (
+                "id"	INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+                "user_id" INTEGER NOT NULL,
+                "log_id" INTEGER NOT NULL,
+                "is_read" BOOLEAN NOT NULL,
+                "message" TEXT NOT NULL)'
+
+    
+    db.execute 'CREATE TABLE logs (
                 "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
-                "username" VARCHAR(255) UNIQUE NOT NULL,
-                "encrypted_password" VARCHAR(255) NOT NULL)'
-
-    encrypted_password = BCrypt::Password.create("123")
-    db.execute('INSERT INTO users (username, encrypted_password) VALUES (?, ?)', "Linus", encrypted_password)
-    encrypted_password = BCrypt::Password.create("321")
-    db.execute('INSERT INTO users (username, encrypted_password) VALUES (?, ?)', "Sunil", encrypted_password)
-    20.times do 
-      title = Faker::Lorem.words(number: 2..5).join(' ').capitalize
-      description = Faker::Lorem.paragraph(sentence_count: 1..3)
-      isCompleted = rand(0..1)
-      db.execute('INSERT INTO todos (title, description, isCompleted, user_id) VALUES (?,?,?, ?)', title, description, isCompleted, 1)
-    end
+                "student_id" INTEGER NOT NULL,
+                "created_at" DATETIME DEFAULT CURRENT_TIMESTAMP,
+                "last_commented_at" DATETIME DEFAULT CURRENT_TIMESTAMP,
+                "is_viewed" BOOLEAN NOT NULL,
+                "done" TEXT NOT NULL,
+                "learned" TEXT NOT NULL,
+                "hard" TEXT NOT NULL,
+                "more" TEXT NOT NULL
```

Comment:
Bör inte dessa kunna vara tomma? Säg att man har lite tid och inte hinner skriva allting. Ska databasen rejecta att skapa den nya loggen? Jag tänker att det är bättre att frontenden ser att du inte skrivit klart allt och sedan promptar en fråga om man verkligen vill lämna in loggen ofärdig. Vad tycker du?

```suggestion
                "more" TEXT
```

Reply:
Jag tänker att om man ser till att frontenden alltid skickar en sträng. No input = "".
Annars måste vi hela tiden kolla om värdet e null.